tests/nosetests/pyanaconda_tests/module_blivet_test.py

```diff
@@ -101,7 +101,7 @@ def send_request_test(self):
         self.assertIsInstance(answer, ProxyID)
         self.assertEqual(answer.id, 0)
 
-    @patch('pyanaconda.dbus.DBus.publish_object')
+    @patch_publisher
```

Comment:
Please, rename it to `patch_dbus_publish_object`, so it is clear what the publisher is. Otherwise, it is just confusing.

Reply:
Fixed.